### PR TITLE
Issue #1131

### LINKED ISSUE #1131: Implement validator/application cache
**Is your feature request related to a problem? Please describe.**
Simple, configurable LRU validator and application cache

### PR DESCRIPTION
**Pull request by oten91**
Closes #1131

### CODE REVIEW

**Review by andrewnguyen22 — DISMISSED** [review]
don't use cosmos implementation

**Comment by oten91** [comment]
Using LRU Cache now

File: types/lru.go [hunk 1]
```diff
@@ -0,0 +1,120 @@
+package types
+
+import (
+	"github.com/hashicorp/golang-lru/simplelru"
+)
+
+var CacheStore *Cache = NewCache(20)
```
Comment (andrewnguyen22) [inline types/lru.go]: Told ed to delete this, just incase you should delete it to

**Review by andrewnguyen22 — DISMISSED** [review]

File: types/lru.go (hunk 1 above)
Comment (oten91) [reply types/lru.go]: removed!

**Review by andrewnguyen22 — APPROVED** [review]

**Pull request merged**